### PR TITLE
Add authentik configuration example

### PR DESCRIPTION
**Pull request by arcoast**
This should import users, groups & memberships

### CODE REVIEW

**Comment by arcoast** [comment]
Sorry, I'm an idiot, been playing with `yaml` files for the last week and just muscle memoried that as yaml. :man_facepalming:

File: example_configs/authentik.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- [x] Sync groups
+
+# Connection settings
+
+## Server URI
+```
+ldap://lldap:3890
+```
+
+- [ ] Enable StartTLD
```
Comment (nitnelave) [inline example_configs/authentik.md:22]: StartTLS ?

File: example_configs/authentik.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+## Base DN
+```
+dc=example,dc=com
+```
+
+# LDAP Attribute mapping
+## User Property Mappings 
+- [x] authentik default LDAP Mapping: mail
+- [x] authentik default LDAP Mapping: Name
+- [ ] authentik default Active Directory Mapping: givenName
```
Comment (nitnelave) [inline example_configs/authentik.md:48]: You can probably tick givenName and sn too

File: example_configs/authentik.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+- [ ] authentik default Active Directory Mapping: sAMAccountName
+- [ ] authentik default Active Directory Mapping: sn
+- [ ] authentik default Active Directory Mapping: userPrincipalName
+- [x] authentik default OpenLDAP Mapping: cn
+- [x] authentik default OpenLDAP Mapping: uid
+
+## Group Property Mappings
+- [x] authentik default LDAP Mapping:
```
Comment (nitnelave) [inline example_configs/authentik.md:56]: Missing the field names?

File: example_configs/authentik.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+## Group membership field
+```
+member
+```
+
+## Object uniqueness field
+```
+cn
```
Comment (nitnelave) [inline example_configs/authentik.md:104]: uid

**Review by nitnelave — CHANGES_REQUESTED** [review]
Looks good! A couple of nits here and there.

**Comment by arcoast** [comment]
I'm not going to lie, I cringed this morning before you responded at the awful quality of this PR.  I feel like it's the Github equivalent of waking up in the morning after a heavy night of drinking to find you've texted an ex.  Three commits, a ton of mistakes and as soon as I saw it again this morning I felt regret and shame! :smile: 

I can only offer my sincerest apologies!

I've corrected the issues above, the only question I've got is whether (now I've added the field names for the `Group property mappings` whether `sn` should be selected the same as you've suggested for the `User property mappings`

File: example_configs/authentik.md (hunk 4 above)
Comment (arcoast) [reply example_configs/authentik.md:104]: Fixed

File: example_configs/authentik.md (hunk 3 above)
Comment (arcoast) [reply example_configs/authentik.md:56]: > Missing the field names?

Added the field names, do you think `sn` should be selected in this section too?

File: example_configs/authentik.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+- [x] authentik default LDAP Mapping: Name
+- [x] authentik default Active Directory Mapping: givenName
+- [ ] authentik default Active Directory Mapping: sAMAccountName
+- [x] authentik default Active Directory Mapping: sn
+- [ ] authentik default Active Directory Mapping: userPrincipalName
+- [x] authentik default OpenLDAP Mapping: cn
+- [x] authentik default OpenLDAP Mapping: uid
+
+## Group Property Mappings
+- [x] authentik default LDAP Mapping: mail
```
Comment (nitnelave) [inline example_configs/authentik.md:56]: Groups only have a display name (aka uid) and members, we can remove mail .

Actually, let's only keep uid

**Comment by arcoast** [comment]
So, spent tonight doing a lot of testing on fresh installs of Authentik.

Most of the config is working well, however the `Group property mappings` is where I've narrowed the issue down to.

```md
## Group Property Mappings
- [ ] authentik default LDAP Mapping: mail
- [ ] authentik default LDAP Mapping: Name
- [ ] authentik default Active Directory Mapping: givenName
- [ ] authentik default Active Directory Mapping: sAMAccountName
- [ ] authentik default Active Directory Mapping: sn
- [ ] authentik default Active Directory Mapping: userPrincipalName
- [ ] authentik default OpenLDAP Mapping: cn
- [x] authentik default OpenLDAP Mapping: uid
```
is what I think you've advised above, but if I set the parameters up with that then run a sync, this is the output I get:
```bash
{"event": "Loaded config", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333965.7756226, "file": "/authentik/lib/default.yml"}
{"event": "Loaded environment variables", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333965.7757845, "count": 7}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5909514, "path": "authentik.admin.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5916529, "path": "authentik.crypto.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5921469, "path": "authentik.events.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5932038, "path": "authentik.outposts.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.594619, "path": "authentik.policies.reputation.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5956225, "path": "authentik.providers.saml.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.596416, "path": "authentik.sources.ldap.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5970845, "path": "authentik.sources.plex.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5979536, "path": "authentik.stages.authenticator_static.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5982623, "path": "authentik.stages.authenticator_totp.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.5985699, "path": "authentik.stages.authenticator_validate.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333966.6015577, "path": "authentik.blueprints.settings"}
{"event": "Booting authentik", "level": "info", "logger": "authentik.lib.config", "timestamp": 1676333966.6016572, "version": "2023.1.2"}
{"event": "Loaded GeoIP database", "last_write": 1674481253.0, "level": "info", "logger": "authentik.events.geo", "pid": 7, "timestamp": "2023-02-14T00:19:26.950073"}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "f591b73e-a01f-456e-8b87-2d0a3af1ed6d", "task_name": "authentik.blueprints.v1.tasks.blueprints_discover", "timestamp": "2023-02-14T00:19:27.516661"}
{"cls": "<class 'authentik.sources.ldap.sync.users.UserLDAPSynchronizer'>", "event": "Starting sync", "level": "info", "logger": "authentik.sources.ldap.management.commands.ldap_sync", "pid": 7, "timestamp": "2023-02-14T00:19:28.012115"}
{"cls": "<class 'authentik.sources.ldap.sync.groups.GroupLDAPSynchronizer'>", "event": "Starting sync", "level": "info", "logger": "authentik.sources.ldap.management.commands.ldap_sync", "pid": 7, "timestamp": "2023-02-14T00:19:28.319065"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=wiki,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'wiki'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.470576", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "57523683-10d1-46b9-83c0-411866564fb0", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.479004"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=synapse,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'synapse'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.481591", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "62fc4d08-4693-44f8-ba2e-70d30049b2a3", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.490271"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=photos,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'photos'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.502787", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "57dabd44-ebfc-4dbb-9a96-9f4dcfcc3f72", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.508918"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=nextcloud,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'nextcloud'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.511349", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "ea806fa9-2d3d-4a75-a5b1-108f35ece1ca", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.518059"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=lldap_strict_readonly,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'lldap_strict_readonly'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.520646", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "9ea1065a-e770-4c00-9b03-2287bdc6d3b5", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.527663"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=lldap_password_manager,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'lldap_password_manager'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.530369", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "89c4c406-eee2-449d-ac2e-a2d1e185e9ff", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.539507"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=lldap_admin,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'lldap_admin'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.542192", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "515c2157-12d7-4072-a704-7b21b6c50f17", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.550721"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=git,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'git'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.563066", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "517033b6-cc49-4bdd-a6e9-1dbc2c3974f4", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.569085"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=emby,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'emby'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.571647", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "66455f1d-4a15-4d4a-8013-469ff31525e2", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.577417"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=calibreweb,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'calibreweb'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.579700", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "bfe41d8d-cda0-4c24-abdd-e009a0816d9b", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.588543"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=airsonic,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'airsonic'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.591712", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "2f2cf996-3718-4b26-b892-9af6af3292b1", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.600639"}
{"action": "configuration_error", "client_ip": null, "context": {"dn": "cn=admin,ou=groups,dc=example,dc=com", "message": "Failed to create group: Name was not set by propertymappings To merge new group with existing group, set the groups's Attribute 'ldap_uniq' to 'admin'", "source": {"app": "authentik_sources_ldap", "model_name": "ldapsource", "name": "lldap", "pk": "3fccd0566b9646519a51ff3a0006ad1d"}}, "event": "Created Event", "level": "info", "logger": "authentik.events.models", "pid": 7, "timestamp": "2023-02-14T00:19:28.603701", "user": {}}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "0cc3fa73-ce46-4df2-a4b6-338df695468e", "task_name": "authentik.events.tasks.event_notification_handler", "timestamp": "2023-02-14T00:19:28.611630"}
{"cls": "<class 'authentik.sources.ldap.sync.membership.MembershipLDAPSynchronizer'>", "event": "Starting sync", "level": "info", "logger": "authentik.sources.ldap.management.commands.ldap_sync", "pid": 7, "timestamp": "2023-02-14T00:19:28.612241"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.722210"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.722889"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.723449"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.724026"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.724573"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.725111"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.725618"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.726133"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.726632"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.727142"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.727691"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.728217"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.728733"}
{"event": "Group does not exist in our DB yet, run sync_groups first: '[]'", "group": [], "level": "warning", "logger": "authentik.sources.ldap.sync.base", "pid": 7, "source": "<LDAPSource: lldap>", "syncer": "MembershipLDAPSynchronizer", "timestamp": "2023-02-14T00:19:28.729253"}
```

However if I set the `Group property mappings` using `cn`
```md
## Group Property Mappings
- [ ] authentik default LDAP Mapping: mail
- [ ] authentik default LDAP Mapping: Name
- [ ] authentik default Active Directory Mapping: givenName
- [ ] authentik default Active Directory Mapping: sAMAccountName
- [ ] authentik default Active Directory Mapping: sn
- [ ] authentik default Active Directory Mapping: userPrincipalName
- [x] authentik default OpenLDAP Mapping: cn
- [ ] authentik default OpenLDAP Mapping: uid
```
then the groups are correctly imported into Authentik
```bash
root@lantallack:/mnt/disks/virtualisation/appdata# ldapsync
{"event": "Loaded config", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.0435355, "file": "/authentik/lib/default.yml"}
{"event": "Loaded environment variables", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.043718, "count": 7}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9175599, "path": "authentik.admin.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9182546, "path": "authentik.crypto.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9187317, "path": "authentik.events.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9197915, "path": "authentik.outposts.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9212196, "path": "authentik.policies.reputation.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9222243, "path": "authentik.providers.saml.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9230103, "path": "authentik.sources.ldap.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9236593, "path": "authentik.sources.plex.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9245346, "path": "authentik.stages.authenticator_static.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.924839, "path": "authentik.stages.authenticator_totp.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.925139, "path": "authentik.stages.authenticator_validate.settings"}
{"event": "Loaded app settings", "level": "debug", "logger": "authentik.lib.config", "timestamp": 1676333643.9281616, "path": "authentik.blueprints.settings"}
{"event": "Booting authentik", "level": "info", "logger": "authentik.lib.config", "timestamp": 1676333643.9282498, "version": "2023.1.2"}
{"event": "Loaded GeoIP database", "last_write": 1674481253.0, "level": "info", "logger": "authentik.events.geo", "pid": 7, "timestamp": "2023-02-14T00:14:04.301764"}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 7, "task_id": "0be970b7-4d0e-4fbf-8204-6ff06fe7d99c", "task_name": "authentik.blueprints.v1.tasks.blueprints_discover", "timestamp": "2023-02-14T00:14:04.896179"}
{"cls": "<class 'authentik.sources.ldap.sync.users.UserLDAPSynchronizer'>", "event": "Starting sync", "level": "info", "logger": "authentik.sources.ldap.management.commands.ldap_sync", "pid": 7, "timestamp": "2023-02-14T00:14:05.379733"}
{"cls": "<class 'authentik.sources.ldap.sync.groups.GroupLDAPSynchronizer'>", "event": "Starting sync", "level": "info", "logger": "authentik.sources.ldap.management.commands.ldap_sync", "pid": 7, "timestamp": "2023-02-14T00:14:05.690700"}
{"cls": "<class 'authentik.sources.ldap.sync.membership.MembershipLDAPSynchronizer'>", "event": "Starting sync", "level": "info", "logger": "authentik.sources.ldap.management.commands.ldap_sync", "pid": 7, "timestamp": "2023-02-14T00:14:05.958582"}
```

Importing groups and members does seem to be where the issues predominantly arise in Authentik, so I'm not sure if the issue is Authentik not mapping the LLDAP parameters as expected. 

Happy to change the PR to reflect what you think best or if you want me to do any more testing then let me know and I can hop in Discord and discuss things more.

**Comment by nitnelave** [comment]
Sounds good, let's keep the `cn` attribute for groups. Thanks!

**Review by nitnelave — APPROVED** [review]

File: example_configs/authentik.md (hunk 3 above)
Comment (nitnelave) [reply example_configs/authentik.md]: No, just `cn`.

**Pull request merged**